HolidayList/Pages/Index.cshtml.cs

```diff
@@ -13,67 +13,77 @@ namespace HolidayList.Pages
                 Places[] Place = QuickTypePlaces.Places.FromJson(Places);
-                
-                QuickType.Welcome welcome = QuickType.Welcome.FromJson(jsonString);
-                List <QuickType.Holiday> holidays = new List<Holiday>();
-                string state = null;
+
+                QuickType.Welcome welcome = QuickType.Welcome.FromJson(CalenderData);
+                List<QuickType.Holiday> holidays = new List<Holiday>();
+                string State = null;
                 foreach (QuickTypePlaces.Places Area in Place)
                 {
-                    if(Area.Country == "United States" && Area.Subcountry == "New York")
+                    if (Area.Country == "United States" && Area.Subcountry == "New York")
                     {
 
-                        state = Area.Subcountry;
+                        State = Area.Subcountry;
                         break;
                     }
                 }
 
 
                 foreach (QuickType.Holiday Holidays in welcome.Response.Holidays)
                 {
-                   if (Holidays.States.Enum == null)
+                    if (Holidays.States.Enum == null)
                     {
-                        if ((Holidays.States.StateArray[0].Name) == state)
+                        if ((Holidays.States.StateArray[0].Name) == State)
                         {
                             holidays.Add(Holidays);
                         }
-                         
+
                     }
                     else
                     {
                         holidays.Add(Holidays);
                     }
-                   
+
                 }
 
                 ViewData["Holidays"] = holidays;
-                ViewData["state"] = state;
+                ViewData["state"] = State;
                 ViewData["Places"] = Place;
 
                 String filterid = null;
                 foreach (QuickType.Holiday Holidays in welcome.Response.Holidays)
                 {
                     if (Holidays.States.Enum == null)
                     {
-                        if ((Holidays.States.StateArray[0].Name) == state)
+                        if ((Holidays.States.StateArray[0].Name) == State)
                         {
                             filterid = Holidays.States.StateArray[0].Iso;
                         }
                     }
                 }
 
-                String filteredHolidaysList = webClient.DownloadString("https://calendarific.com/api/v2/holidays?&api_key=3b2cc4a9fcb58a121947815135fca92694b0f63e&country=US&year=2019&location="+ filterid);
+                String filteredHolidaysList = DownloadData("https://calendarific.com/api/v2/holidays?&country=US&year=2019&api_key=" + CalenderAPIkey + "&location=" + filterid);
+
                 QuickType.Welcome StateHoliday = QuickType.Welcome.FromJson(filteredHolidaysList);
 
-                return new JsonResult(StateHoliday);
+                // return new JsonResult(StateHoliday);
```

Comment:
Don't commit commented out code.   Just delete it.  We can always go back in history in version control to see what used to be there.

Also... the group may want to use an HTML page instead of a JSON result...  Or, perhaps they can make the JSON result in a different view.